.github/workflows/release.yaml

```diff
@@ -238,38 +278,43 @@ jobs:
           - 'v1.23.6'
           - 'v1.24.2'
           - 'v1.25.3'
+        test: ${{ fromJSON(needs.setup-e2e-tests.outputs.test_names) }}
```

Comment:
Just to note that this will produce N (number of k8s versions) times M ( number of e2e tests) jobs in CI. Fine by me but we might monitor the runtime of this pipeline next time we trigger a release